_toc.yml

```diff
@@ -107,6 +107,8 @@ entries:
               title: Apache Kafka and OpenSearch
             - file: docs/tools/terraform/reference/cookbook/multicloud-postgresql-recipe
               title: Multicloud PostgreSQL 
+            - file: docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe
+              title: Add the power of Apache Flink® to Apache Kafka® for stateful stream processing  
```

Comment:
This title should be shorter and exclude the trademarks, so that it displays well in the nav.  "Apache Kafka and Apache Flink" or something along those lines could work.